### ISSUE TITLE
Ansible 2.5 slice filter error

### ISSUE THREAD

**Issue by dobrozhanskyi** [ahuffman/ansible-sudoers#24]
TASK [ansible-sudoers : Ensure the sudoers file is valid and up to date (separate specs)] ********************************************************************************
fatal: [mngt.us.dcapi.net]: FAILED! => {"changed": false, "msg": "AnsibleFilterError: The slice filter requires a mapping to operate on, got a <type 'list'>."}

**Comment by DeLoWaN** [comment]
Same here. @dobrozhanskyi got this using python2.

With python3.5.2, I get this kind of error:
```
TASK [ansible-sudoers : Ensure the sudoers file is valid and up to date (separate specs)] ***************************
fatal: [preprod.iwd.fr]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iteritems'"}
```

I tried with different versions of ansible from 2.3 to 2.5 with the same results. Do you guys have any idea?

Note that all was fine with ansible 2.4.2.

**Comment by kuch4r** [comment]
There is issue for this in Ansible:
https://github.com/ansible/ansible/issues/37890

**Comment by CafeLungo** [comment]
I am now running into this, on Ansible 2.5.0

**Comment by sebPomme** [comment]
You can uptdate your ansible version the slice bug is fixed in version 2.5.2 https://github.com/ansible/ansible/pull/37944 (its work for me)

**Comment by ahuffman** [comment]
Fixed with Ansible 2.5.2.  Thanks for validating @sebPomme and mentioning the Ansible core issue @kuch4r .  Appreciate it.  :1st_place_medal:
